src/boot/efi/stub.c

```diff
@@ -482,17 +482,19 @@ static EFI_STATUS run(EFI_HANDLE image) {
                 log_error_status(err, "Error loading global addons, ignoring: %m");
         parameters_measured = parameters_measured < 0 ? m : (parameters_measured && m);
 
-        _cleanup_free_ char16_t *dropin_dir = get_extra_dir(loaded_image->FilePath);
-        err = cmdline_append_and_measure_addons(
-                        image,
-                        loaded_image,
-                        dropin_dir,
-                        uname,
-                        &m,
-                        &cmdline);
-        if (err != EFI_SUCCESS)
-                log_error_status(err, "Error loading UKI-specific addons, ignoring: %m");
-        parameters_measured = parameters_measured < 0 ? m : (parameters_measured && m);
+        if (loaded_image->FilePath) {
```

Comment:
can you add a comment here: `/* handle case where not backed by proper file system */` or so?